src/app/shell/provider/provider-activities/create-activity/create-about-form/create-about-form.component.ts

```diff
@@ -1,62 +1,43 @@
-import { Component, EventEmitter, OnInit, Output } from '@angular/core';
-import { FormBuilder, FormControl, FormGroup } from '@angular/forms';
-import { Constants } from '../../../../../shared/constants/constants';
+import { Component, ElementRef, EventEmitter, OnInit, Output, ViewChild } from '@angular/core';
+import { FormArray, FormBuilder, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-create-about-form',
   templateUrl: './create-about-form.component.html',
-  styleUrls: ['./create-about-form.component.scss','./../../../validation.component.scss']
+  styleUrls: ['./create-about-form.component.scss', './../../../validation.component.scss']
 })
 export class CreateAboutFormComponent implements OnInit {
 
   radioBtn = new FormControl(false);
   priceCtrl = new FormControl({ value: 0, disable: true });
   AboutFormGroup: FormGroup;
   @Output() PassAboutFormGroup = new EventEmitter();
+  @ViewChild('maxAgeInput') maxAgeInput: ElementRef<HTMLInputElement>;
+  @ViewChild('minAgeInput') minAgeInput: ElementRef<HTMLInputElement>;
 
   constructor(private formBuilder: FormBuilder) {
     this.AboutFormGroup = this.formBuilder.group({
+      title: new FormControl('', Validators.required),
       type: new FormControl(''),
-      title: new FormControl(''), 
-      phone: new FormControl(''),
-      email: new FormControl(''),
-      ageFrom: new FormControl(''),
-      ageTo: new FormControl(''),
-      classAmount: new FormControl(''),
+      phone: new FormControl('', [Validators.required, Validators.maxLength(9), Validators.minLength(9)]),
+      email: new FormControl('', [Validators.required, Validators.email]),
+      minAge: new FormControl('', [Validators.required, Validators.maxLength(2)]),
```

Comment:
as a suggestion, you can try Validators.max(16), Validators.min(1) also

Reply:
I don't know why, but it didn't work:/ 
should I try more and get rid of the directive?